foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/PoetryMigration.java

```diff
@@ -0,0 +1,118 @@
+package com.boozallen.aissemble.upgrade.migration;
```

Comment:
Can move this to the `v1_7_0` package/directory since it's only needed for this one upgrade. Same with the test files and spec.

Reply:
Moved.